nautobot/docs/user-guide/administration/installation/app-install.md

```diff
@@ -11,7 +11,7 @@ You must be **absolutely** sure to install the app within Nautobot's virtual env
 
 Download and install the app package per its installation instructions. Apps published via PyPI are typically installed using `pip3`.
 
-```no-highlight
+```no-highlight title="Pip install the package"
 pip3 install <package>
 ```
 
```

Comment:
Let's remove the lines below about `python setup.py` as they don't apply to the majority of Apps these days.